### ISSUE TITLE
"OpenTabletDriver Tilt Broken in Krita (XWayland)

### ISSUE THREAD

**Issue by foldfree** [pop-os/cosmic-comp#1056]
When using OpenTabletDriver, my graphic tablet works except for tilt, which is not detected in Krita (XWayland).
In Gnome DE, tilt works as expected.

![Screenshot with tablet debugging on Krita and OTD](https://github.com/user-attachments/assets/d8427f14-75ce-4d09-a28f-8d5c64808bf7)
About the screeshot:
On the left is Krita's Tablet Tester: the tilt values (TX and TY) are stuck at 0.
On the right is OTD's Tablet debugger: tilt is properly detected.

**Comment by McArthur-Alford** [comment]
I'm running into the exact same issue on my artist pro 16 (gen 2), so adding that to the record in case hardware is relevant.

**Comment by homeless-field** [comment]
Is this issue still present? I'm having problems getting pressure to work and I see you have it here. Are you in artist mode, and what's your version number?

**Comment by foldfree** [comment]
Yes, in artist mode. OPT is 0.6.5.1 and cosmic alpha 7 from nixpkg unstable.
